### PR TITLE
Plan for slowest key press task

### PR DESCRIPTION
**Pull request by Jolita-byte**
modified Readme file by adding implementation plan at the bottom

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,4 +62,14 @@ The longest of these was the keypress for 'a' with duration 16.
 - 2 <= n <= 1000
 - 1 <= releaseTimes[i] <= 109
 - releaseTimes[i] < releaseTimes[i+1]
-- keysPressed contains only lowercase English letters.
\ No newline at end of file
+- keysPressed contains only lowercase English letters.
+
+###Implementation plan
```
Comment (Almantask) [inline README.md]: Space between ### and the rest

File: README.md [hunk 2]
```diff
@@ -62,4 +62,14 @@ The longest of these was the keypress for 'a' with duration 16.
...
+- If keysPressed String is null or empty, return INVALID;
+- If releaseTimes is null, return INVALID;
+- If releaseTimes is not the same length as keysPressed, return INVALID;
+- Find durations for each key press. Store them in separate array;
+- Find the largest number in durations array. This number is the slowest key press time;
```
Comment (Almantask) [inline README.md]: Use 1. 2. 3. etc instead

**Review by Almantask — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Jolita-byte) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Jolita-byte) [reply README.md]: done

**Review by Almantask — APPROVED** [review]
👍

**Pull request merged**